src/ruby/pb/generate_proto_ruby.sh

```diff
@@ -40,7 +42,13 @@ $PROTOC -I . \
     --plugin=$PLUGIN
 
 $PROTOC -I . \
-    src/proto/grpc/testing/{messages,payloads,stats,benchmark_service,report_qps_scenario_service,worker_service,control}.proto \
+    src/proto/grpc/testing/messages.proto \
```

Comment:
any reason why this change or is this just refactoring?

Reply:
For some reason, that syntax (`{messages,test,empty}.proto`) sometimes doesn't work for me. Ran into some sort of syntax error when I run the original script unmodified. But I couldn't reproduce that anymore. I have reverted this change.